api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/ResumoMensalQueries.java

```diff
@@ -37,11 +37,132 @@ public static String getAllPatientsWithPreArtStartDateLessThanReportingStartDate
+            + "               AND o.concept_id =${conceptId}"
+            + "        UNION ALL "
+            + "        SELECT p.patient_id, "
+            + "               date_enrolled AS enrollment_date "
+            + "        FROM   patient_program pp "
+            + "               JOIN patient p "
+            + "                 ON pp.patient_id = p.patient_id "
+            + "        WHERE  p.voided = 0 "
+            + "               AND pp.voided = 0 "
+            + "               AND pp.program_id =${programId} "
+            + "               AND pp.location_id =:location "
+            + "               AND pp.date_enrolled BETWEEN :startDate AND :endDate "
+            + "        UNION ALL "
+            + "        SELECT p.patient_id, "
+            + "               enc.encounter_datetime AS enrollment_date "
+            + "        FROM   encounter enc "
+            + "               JOIN patient p "
+            + "                 ON p.patient_id = enc.patient_id "
+            + "        WHERE  p.voided = 0 "
+            + "               AND enc.encounter_type IN (${encounterType1},${encounterType2}) "
+            + "               AND enc.location_id =:location "
+            + "               AND enc.encounter_datetime BETWEEN"
+            + "                   :startDate AND :endDate "
+            + "        ORDER  BY enrollment_date ASC) results "
+            + "GROUP  BY results.patient_id) res  ";
+
+    StringSubstitutor stringSubstitutor = new StringSubstitutor(map);
+    return stringSubstitutor.replace(query);
+  }
+
+  /**
+   * All patients Enrolled in Pre-Art or Registered In Clinical Process Opening enddate
+   *
+   * @return String
+   */
+  public static String getAllPatientsRegisteredAsTransferredInProgramEnrolmentWithBoundaries(
+      int stateId, int programId) {
+
+    Map<String, Integer> map = new HashMap<>();
+    map.put("stateId", stateId);
+    map.put("programId", programId);
+    String query =
+        "SELECT p.patient_id "
+            + "FROM  patient p "
+            + "       INNER JOIN patient_program pp "
+            + "               ON p.patient_id = pp.patient_id "
+            + "       INNER JOIN patient_state ps "
+            + "               ON ps.patient_program_id = pp.patient_program_id "
+            + "WHERE  p.voided = 0 "
+            + "       AND ps.state = ${stateId} "
+            + "       AND pp.location_id =:location "
+            + "       AND pp.date_enrolled BETWEEN :startDate AND :endDate "
+            + "       AND pp.program_id = ${programId} ";
+
+    StringSubstitutor stringSubstitutor = new StringSubstitutor(map);
+    return stringSubstitutor.replace(query);
+  }
+
+  /**
+   * Number of patients transferred-in from another HFs during the current month
+   *
+   * @return String
+   */
+  public static String getPatientsTransferredFromAnotherHealthFacilityDuringTheCurrentMonth(
```

Comment:
This method should be deleted because there are no usages.

Reply:
Hey @ynurmahomed , i'm not the one who implemented that method... that's why i prefered to not mess with it! But im going to remove it now!

Reply:
@ynurmahomed no problem in removing, unused code should never be commited.